### PR TITLE
Task #217140 : content detail Page API integration

### PR DESCRIPTION
**Pull request by mahajanmahesh935**
## Description

Please describe the changes in your pull request in few words here.

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

Describe the steps required to test the changes proposed in the pull request.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

File: packages/nulp_elite/src/pages/search/ContentList.js [hunk 1]
```diff
@@ -110,22 +109,21 @@ const ContentList = (props) => {
...
+    console.log("RandomImage   --  ", RandomImage.ImagePaths);
+    const randomIndex = RandomImage;
+    // const randomIndex = Math.floor(Math.random() * RandomImage..length);
+    console.log("randomIndex", randomIndex);
+
+    // return array[randomIndex];
+    return randomIndex;
+  };
 
-    navigate('/contentList/'+value, { state: { domain:domain }})
```
Comment (paritshivani) [inline packages/nulp_elite/src/pages/search/ContentList.js:128]: Dont revert this changes, will affect on functionality

**Comment by mahajanmahesh935** [comment]
snehal asked to Comment Button in addConnection.js

File: packages/nulp_elite/src/pages/search/ContentList.js (hunk 1 above)
Comment (paritshivani) [reply packages/nulp_elite/src/pages/search/ContentList.js:128]: navigate('/contentList/'+value, { state: { domain:domain }})
